### PR TITLE
[READY]Flightsuit Fix

### PR DESCRIPTION
**Pull request by silicons**
There's also a bug with them phasing past airlocks that I havn't been able to reproduce yet.
:cl:
bugfix: Flightsuits actually fly over people
bugfix: Flightsuits don't interrupt pulls when you pass through doors
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
Wasn't airlock phasing intended?

**Comment by kevinz000** [comment]
not if the airlock shouldn't be opening.

**Comment by Jordie0608** [comment]
Did you test this?

File: code/modules/clothing/spacesuits/flightsuit.dm [hunk 1]
```diff
@@ -263,11 +263,17 @@
...
 				losecontrol()
 			momentum_decay()
+			var/atom/movable/pull = wearer.pulling
 			for(var/i in 1 to momentum_speed)
+				var/turf/oldturf = get_turf(wearer)
 				if(momentum_speed_x >= i)
 					step(wearer, drift_dir_x)
 				if(momentum_speed_y >= i)
 					step(wearer, drift_dir_y)
+				if(pull)
```
Comment (optimumtact) [inline code/modules/clothing/spacesuits/flightsuit.dm]: just combine these two if's

**Comment by kevinz000** [comment]
This works to an extent which makes further improvements unnecessary.
as in it works(tm)

**Pull request merged**